eclair-core/src/test/scala/fr/acinq/eclair/integration/basic/fixtures/MinimalNodeFixture.scala

```diff
@@ -282,9 +282,23 @@ object MinimalNodeFixture extends Assertions with EitherValues {
     val sender = TestProbe("sender")
     sender.send(node2.paymentHandler, MultiPartHandler.ReceivePayment(Some(amount), Left("test payment")))
     val invoice = sender.expectMsgType[Bolt11Invoice]
+    val invoiceWithHints = Bolt11Invoice(
+      chainHash = node2.nodeParams.chainHash,
+      amount = invoice.amount_opt,
+      paymentHash = invoice.paymentHash,
+      privateKey = node2.nodeParams.privateKey,
+      description = invoice.description,
+      minFinalCltvExpiryDelta = invoice.minFinalCltvExpiryDelta,
+      fallbackAddress = invoice.fallbackAddress(),
+      expirySeconds = invoice.tags.collectFirst { case expiry: Bolt11Invoice.Expiry => expiry.toLong },
+      extraHops = hints.map(_.toList).toList,
+      timestamp = invoice.createdAt,
+      paymentSecret = invoice.paymentSecret.get,
+      paymentMetadata = invoice.paymentMetadata,
+      features = invoice.features)
```

Comment:
I missed this, can you fix in #2350?

```suggestion
    assert(invoice.routingInfo.isEmpty)
    val invoiceWithHints = invoice
      .modify(_.tags).using(_ ++ hints.map(hops => Bolt11Invoice.RoutingInfo(hops.toList)))
```